apps/zotonic_launcher/src/command/zotonic_cmd_config.erl

```diff
@@ -173,15 +178,26 @@ pretty_print_value(_Key, Value) when is_map(Value) ->
                 end;
             ({K, V}) when is_number(V); is_boolean(V); is_atom(V) ->
                 io:format("~n      ~p: ~p", [ K, V ]);
+            ({K, {{Y, M, D}, {H, I, S}} = Date}) when
+                is_integer(Y), M >= 1, M =< 12, D >= 1, D =< 31,
```

Comment:
`>` comparisons on integers can sometimes give surprising results in erlang. Especially when you after this comparison expect values to be integer.

For example:
```erlang
(zotonic@localhost)10> undefined >= 1.
true
```